pkg/plugin/linuxutil/netstat_stats_linux.go

```diff
@@ -269,36 +245,12 @@ func (nr *NetstatReader) updateMetrics() {
 		metrics.TCPStateGauge.WithLabelValues(state).Set(float64(v))
 	}
 
-	for remoteAddr, v := range nr.connStats.TcpSockets.socketByRemoteAddr {
-		addrPort, err := netip.ParseAddrPort(remoteAddr)
-		if err != nil {
-			nr.l.Error("Failed to parse remote address", zap.Error(err))
```

Comment:
let's keep this error check for the current socket stats?

Reply:
we still have that check. It's just for previous stats instead of current stats

Reply:
error check added.